### PR TITLE
Removing default admin status initialization in DPB flow while loading minigraph

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

#### Description for the changelog

The config load_minigraph initializes 'admin_status' to up when platform.json has DPB configs. This doesn't happen when using port_config.ini
The update minigraph has logic to initialize only the ports whose neighbors are defined or those belonging to portchannel
However, a change was introduced  to have default admin status to be 'up' in portconfig.py when the minigraph was using platform.json

This will lead to sanity check failure in sonic-mgmt and thus no test cases could be run

Fixed the platform.json flow not to initialize admin_status by default. Fixed UT to ensure that different cases are handled.

### CODE REVIEW

File: src/sonic-config-engine/tests/sample_hwsku.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "interfaces": {
         "Ethernet0": {
-            "default_brkout_mode": "1x100G[40G]"
+            "default_brkout_mode": "4x25G[10G]"
```
Comment (keboliu) [inline src/sonic-config-engine/tests/sample_hwsku.json:4]: what's the intention here?

File: src/sonic-config-engine/tests/simple-sample-graph.xml [hunk 2]
```diff
@@ -178,12 +178,12 @@
       <PortChannelInterfaces>
         <PortChannel>
           <Name>PortChannel01</Name>
-          <AttachTo>fortyGigE0/4</AttachTo>
+          <AttachTo>Ethernet4</AttachTo>
```
Comment (keboliu) [inline src/sonic-config-engine/tests/simple-sample-graph.xml:181]: why we change here?  Any functional change by this change? same comments in the below changes in this file.

Comment (dgsudharsan) [reply src/sonic-config-engine/tests/simple-sample-graph.xml:181]: As per the original logic, a port belonging to a port-channel will be admin up. However, the simple-sample-graph file is used for test cases between port_config.ini and platform.json (which is a very poor initial design choice). It was using the alias in port_config.ini. However, these alias were not present in platform.json (which has a defined format of alias). So the common ground between these two is to use the interface name directly.

File: src/sonic-config-engine/tests/sample_hwsku.json (hunk 1 above)
Comment (dgsudharsan) [reply src/sonic-config-engine/tests/sample_hwsku.json:4]: These ports Ethernet1 and Ethernet2 were defined as part of port channel in simple-sample-config.xml. So to align with that I modified these to be in 4x25G[10G] mode.

**Review by keboliu — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Pull request closed without merging**